src/libs/actions/Report.js

```diff
@@ -905,6 +905,7 @@ function editReportComment(reportID, originalReportAction, textForNewComment) {
     // Do not autolink if someone explicitly tries to remove a link from message.
     // https://github.com/Expensify/App/issues/9090
     const htmlForNewComment = parser.replace(textForNewComment, {filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')});
+    const originalMessageHTML = parser.replace(originalReportAction.message[0].html, {filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')});
```

Comment:
`{filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')}` can be DRY up.

Reply:
Good call. Updated!